src/control/README.md

```diff
@@ -32,23 +32,31 @@ For convenience, active parsed config values are written to either directory whe
 
 If user shell executing `daos_server` has environment variable `CRT_PHY_ADDR_STR` set, user os environment will be used when spawning `daos_io_server` instances. In this situation a "Warning: using os env vars..." message will be printed to the console and no environment variables will be added as specified in the `env_vars` list within the per-server section of the server config file. This behaviour provides backward compatibility with historic mechanism of specifying all parameters through environment variables.
 
-## Prerequisites
+## Subcommands
 
-In addition to editing the configuration files prior to starting `daos_server`, an administrator needs to set passwordless sudo for the specific commands listed in `src/control/mgmt/init/setup_spdk.sh` (script is installed as `install/share/setup_spdk.sh`) in order for `daos_server` to enable access to the NVMe SSDs through SPDK when run as an unprivileged user.
+`daos_server` supports various subcommands (see `daos_server --help` for available subcommands) which will perform stand-alone tasks as opposed to launching as a daemon (default operation if launched without subcommand).
 
-The sudoers file can be accessed with command `visudo` and the simplest way to achieve the necessary access is to grant no-password sudo permissions to a user:
-```
-linuxuser ALL=(ALL) NOPASSWD: ALL
-```
-alternatively more fine-grained permissions can be granted e.g.:
+### prep-nvme
+
+This subcommand requires elevated permissions and needs to be run with root permissions (sudo).
+
+NVMe access through SPDK as an unprivileged user can be enabled by first running `sudo daos_server prep-nvme -p 4096`. This will perform the required setup in order for `daos_server` to be granted access to the NVMe SSDs through SPDK when run as an unprivileged user (the user issuing the sudo command) including the allocation of 4096 hugepages.
+
```

Comment:
One thing worth noting is that we probably need to allocate hugepages on boot time in the future, see 2.32 of http://doc.dpdk.org/guides/linux_gsg/sys_reqs.html

Reply:
@johannlombardi should we do this then on reboot when we configure SCM interleaved mode or leave it to configuration management?